app/services/phone_number_capabilities.rb

```diff
@@ -0,0 +1,32 @@
+class PhoneNumberCapabilities
+  VOICE_UNSUPPORTED_US_AREA_CODES = {
+    '648' => 'American Samoa',
+    '671' => 'Guam',
+    '670' => 'Northern Mariana Islands',
+    '340' => 'United States Virgin Islands',
+  }.freeze
+
+  attr_reader :phone
+
+  def initialize(phone)
+    @phone = phone
+  end
+
+  def sms?
+    true
+  end
+
+  def voice?
+    VOICE_UNSUPPORTED_US_AREA_CODES[area_code].nil?
+  end
+
+  def unsupported_location
+    VOICE_UNSUPPORTED_US_AREA_CODES[area_code]
+  end
+
+  private
+
+  def area_code
+    @area_code ||= Phony.split(Phony.normalize(phone, cc: '1')).second
```

Comment:
Personally I think using `Phonelib` here is clearer, like we do here? https://github.com/18F/identity-idp/blob/master/app/forms/otp_delivery_selection_form.rb#L37-L48

```ruby
Phonelib.parse(phone).area_code
```